benchmark/single-source/ObjectiveCBridging.swift

```diff
@@ -914,3 +942,40 @@ public func run_RealNSArrayMutableCopyObjectAtIndex(_ n: Int) {
   #endif
 }
 
+@inline(__always)
+fileprivate func run_BridgedNSStringLength(_ asciiBase: Bool, _ enc: UInt, _ n: Int) {
```

Comment:
These are the same benchmarks as the PR adding benchmarks, so once that merges this part of the diff will go away